volatility3/framework/plugins/yarascan.py

```diff
@@ -13,19 +13,30 @@
 
 vollog = logging.getLogger(__name__)
 
+USE_YARA_X = False
+
 try:
-    import yara
+    import yara_x
+
+    USE_YARA_X = True
 
-    if tuple([int(x) for x in yara.__version__.split(".")]) < (3, 8):
-        raise ImportError
 except ImportError:
-    vollog.info(
-        "Python Yara (>3.8.0) module not found, plugin (and dependent plugins) not available"
-    )
-    raise
+    try:
+        import yara
+
+        if tuple(int(x) for x in yara.__version__.split(".")) < (3, 8):
+            raise ImportError
+
+        vollog.info("Using yara-python module")
+
+    except ImportError:
+        vollog.info(
+            "Python Yara (>3.8.0) module not found, plugin (and dependent plugins) not available"
```

Comment:
This should probably also offer that yara-x can be used as well (so "Neither yara-x not python yara (3.8.0) was found...")...